### DISCUSSION TITLE
How to use Fontawesome Icons Pro

### DISCUSSION THREAD

**Discussion by Reanii** [enisn/UraniumUI#729] [Q&A] [answered]
hello guys, so how can i use icons pro by FontAwesome?

**Comment by enisn** [comment, marked as answer]
Fonts are embedded resources and I can't embed licensed files into library and DLL files. 

In this case, you should;
- Download pro fonts and add `Resources/Fonts` folder of your project.
- Then go to `MauiProgram.cs` and add your fonts like below:
   ```cs
  .ConfigureFonts(fonts =>
  {
      // Font Awesome Pro fonts
      fonts.AddFont("Font Awesome 5 Pro-Solid-900.otf", "FontAwesomeSolid");
      fonts.AddFont("Font Awesome 5 Pro-Regular-400.otf", "FontAwesomeRegular");
      fonts.AddFont("Font Awesome 5 Pro-Light-300.otf", "FontAwesomeLight");
      // etc. ...

  });
   ```

- If you want to generate C# class for **Glyph** characters visit https://andreinitescu.github.io/IconFont2Code/ website and upload your font files to generate C# Poco classes for each icon in the font.

**Reply by Reanii** [reply]
oh nice! thx
